src/app/shared/dockstore.model.ts

```diff
@@ -38,6 +38,7 @@ export class Dockstore {
   static BD_CATALYST_TERRA_IMPORT_URL = 'https://terra.biodatacatalyst.nhlbi.nih.gov/#import-tool/dockstore';
   static CAVATICA_IMPORT_URL = 'https://cavatica.sbgenomics.com/integration/trs/import';
   static NEXTFLOW_TOWER_IMPORT_URL = 'https://tower.nf/launch';
+  static GALAXY_INSTANCE_IMPORT_URL = 'https://usegalaxy.org/workflows/trs_import?trs_server=dockstore.org&trs_id=';
```

Comment:
Not sure why comment was dropped. 
We may need a on/off switch, particularly for DNAstack but also for this depending on https://github.com/dockstore/dockstore/issues/3764#issuecomment-833821786 (is it not working for just versions or overall?)